lxd/db/networks.go

```diff
@@ -42,32 +42,43 @@ func (c *ClusterTx) GetNetworksLocalConfig() (map[string]map[string]string, erro
 // GetNonPendingNetworkIDs returns a map associating each network name to its ID.
```

Comment:
@C-Stout this function's description should have been updated to document that the return value is now a map of pending projects and network names to network IDs. Please can you open another PR?